File: Examples/VisualBasic/Programming-Documents/Fields/FormFieldsGetFormFieldsCollection.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Words
Imports Aspose.Words.Fields
Public Class FormFieldsGetFormFieldsCollection
    Public Shared Sub Run()
        ' ExStart:FormFieldsGetFormFieldsCollection
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_WorkingWithFields()

        Dim doc As New Document(dataDir & Convert.ToString("FormFields.doc"))
        Dim formFields As FormFieldCollection = doc.Range.FormFields

        ' ExEnd:FormFieldsGetFormFieldsCollection
        Console.WriteLine(vbLf & "Document have " + formFields.Count.ToString() + " form fields.")
    End Sub
End Class

File: Examples/CSharp/Programming-Documents/Fields/FormFieldsGetFormFieldsCollection.cs
using System;
using System.Collections;
using System.IO;
using Aspose.Words;
using Aspose.Words.Fields;
namespace Aspose.Words.Examples.CSharp.Programming_Documents.Working_with_Fields
{
    class FormFieldsGetFormFieldsCollection
    {
        public static void Run()
        {
            //ExStart:FormFieldsGetFormFieldsCollection
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_WorkingWithFields();

            Document doc = new Document(dataDir + "FormFields.doc");
            FormFieldCollection formFields = doc.Range.FormFields;

            //ExEnd:FormFieldsGetFormFieldsCollection
            Console.WriteLine("\nDocument have " + formFields.Count.ToString() + " form fields.");
        }
    }
}
